Add OPENBROWSER_AND_MARK to url and item viewformaction

Limero opened the pull request:
This makes user-defined macros with `open-in-browser-and-mark-read` behave the same way as `open-in-browser` for the URL and item/article views instead of being ignored.

I have a macro like `macro r set browser "rtv"; open-in-browser-and-mark-read ; set browser qutebrowser` and before this fix, it's only usable in the article list.

Minoru commented:
This is a tough one. In my opinion, `*-and-mark-read` makes little sense in itemview: if the user is looking at an item, it's obvious that the item is considered read. That's even more true for the urlsview.

On the other hand, I don't have any better ideas on how to resolve this. `macro` doesn't take the dialog name like `bind-key` does, so users can't work around this by writing multiple macros.

If we're going with this approach, let's at least make it a proper part of Newsboat:
1. please update `opdescs` array in the `Keymap` class to make this op available in url- and itemview;
2. please update the docs to explain how this op behaves in the urlview.

Limero commented:
I have updated the opdescs and docs now. I noticed that the regular `open-in-browser` didn't work in the URL view, so I also added that. Let me know if you are ok with this, or if I should scrap open in the URL view for both `open-in-browser` and `open-in-browser-and-mark-read`, or do something differently.
